server/plugin/client.go

```diff
@@ -310,6 +311,20 @@ func (c *client) GetSubscriptionFilterPossibleValues(request *serializers.GetSub
 	return subscriptionFiltersResponse, statusCode, nil
 }
 
+func (c *client) GetApprovalDetails(organization, projectName, mattermostUserID string, approvalID int) (*serializers.PipelineApprovalDetails, int, error) {
+	pipelineApproveRequestURL := fmt.Sprintf(constants.PipelineApproveRequest, organization, projectName, approvalID)
```

Comment:
can't we use this value directly instead of creating a new variable?

Reply:
this is to make it more readable